README.md

```diff
@@ -179,6 +179,38 @@ When freshly starting a node it is usually necessary to also reset Metamask Acco
 <br/><br/>
 
 ---
+## Running dApp in Production/Staging Mode Locally
+There may be a time that you will need to run the dApp in production/staging mode to test out a certain feature or do verification before a deploy. In this case there is no need for a local node as you will connect directly to the mainnet/testnet. 
```

Comment:
To be crystal clear so that external contributors don't get confused, maybe add a note saying this section is only applicable to Origin core team members.

Reply:
I added `Core Contributors` to the header to keep things copacetic.

Reply:
perfect! :)